client/src/contexts/ProductReview.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect, useContext } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { ProductIDContext } from './ProductIDContext';
+
+const ReviewContext = React.createContext();
+ReviewContext.displayName = 'ReviewData';
+
+// custome hook to return useContext
+export function useReviews() {
+  return useContext(ReviewContext);
+}
+
+export function ReviewProvider({ children }) {
+  const productId = useContext(ProductIDContext);
+  const [reviews, setReviews] = useState({});
+
+  useEffect(() => {
+    axios({
+      method: 'get',
+      url: 'http://localhost:3000/reviews',
```

Comment:
You should be able to use `/reviews` without the localhost here. Will save us from having to hunt it down once we start deploying to production

Reply:
Will update